R/learner_Cubist_regr_cubist.R

```diff
@@ -0,0 +1,84 @@
+#' @title Regression Cubist Learner
+#' @author sumny
+#' @name mlr_learners_regr.cubist
+#'
+#' @template class_learner
+#' @templateVar id regr.cubist
+#' @templateVar caller cubist
+#'
+#' @template seealso_learner
+#' @template example
+#' @export
+LearnerRegrCubist = R6Class("LearnerRegrCubist",
+  inherit = LearnerRegr,
+
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      ps = ParamSet$new(
```

Comment:
This needs to be updated in the book but now we're exclusively using paradox sugar for learners (there's still an open PR working through existing learners). So please can you use `ps` syntax, `p_int`, etc.

Reply:
fixed!